src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const li = document.querySelectorAll('li');
+const ul = document.querySelectorAll('ul');
+
+[...li].forEach(el => {
+  const span = document.createElement('span');
+
+  Array.from(ul).forEach(ulEl => {
```

Comment:
It is a bad practice to have two nested loops.
It's actually redundant here, consider having only one loop

Reply:
fix